.github/auto-label.yaml

```diff
@@ -62,3 +62,4 @@ path:
     video-intelligence: "videointelligence"
     vision: "vision"
     workflows: "workflows"
+    dlp: "dlp"
```

Comment:
Nit: recommend keeping the list sorted alphabetically

Reply:
Done